### PR TITLE
Extend Geocode Earth provider from Pelias provider

### PR DESCRIPTION
**Pull request by jbelien**
Related to #1004 (waiting for it to be merged first)

### CODE REVIEW

File: src/Provider/GeocodeEarth/GeocodeEarth.php [hunk 1]
```diff
@@ -52,22 +38,17 @@ public function __construct(HttpClient $client, string $apiKey)
         }
 
         $this->apiKey = $apiKey;
-        parent::__construct($client);
+        parent::__construct($client, 'https://api.geocode.earth/', 1);
```
Comment (atymic) [inline src/Provider/GeocodeEarth/GeocodeEarth.php]: Think it would be good to have these in constants still :)

**Comment by jbelien** [comment]
@atymic I think the GeocoderEarth provider is ready to be upgraded (using the Pelias provider). Could you review it ? :)

**Comment by jbelien** [comment]
There is no breaking changes so I guess we can release this as `1.1.0`, right ?

Ping @Nyholm

**Comment by Nyholm** [comment]
If the Public API did not change than this is fine. 

Why did we need to update all the cached test responses?

**Comment by jbelien** [comment]
> Why did we need to update all the cached test responses?

It was indeed probably not necessary.
Would you prefer me to revert it ?

**Comment by Nyholm** [comment]
Na, thats fine. I just wondered. If we have to rewrite all the tests it may be an indicator that we changed the behaviour.

**Comment by jbelien** [comment]
Just to be sure, I just checked.
Update of the cached response was not necessary indeed.

**Comment by jbelien** [comment]
New release `1.1.0`is live : https://packagist.org/packages/geocoder-php/geocode-earth-provider

**Pull request merged**